### ISSUE TITLE
Add --quiet flag

### ISSUE THREAD

**Issue by fenhl** [nabijaczleweli/cargo-update#115]
It would be nice if this had a `--quiet` flag to only output warnings/errors in the same way as other `cargo` subcommands.

**Comment by nabijaczleweli** [comment]
Can you be a love and see if the commit above works for you?

**Comment by nabijaczleweli** [comment]
Released in [`v2.2.0`](https://github.com/nabijaczleweli/cargo-update/releases/tag/v2.2.0), but I'd still like your take on't.

**Comment by fenhl** [comment]
I am still seeing two lines of output with the `--quiet` flag:

```
    Updating registry 'https://github.com/rust-lang/crates.io-index'

```

**Comment by nabijaczleweli** [comment]
Good catch, should be fixed!

Released in [`v2.2.1`](https://github.com/nabijaczleweli/cargo-update/releases/tag/v2.2.1).
